common/sentinel/src/actors/actor.rs

```diff
@@ -66,7 +65,7 @@ impl TryFrom<&EthLog> for Actor {
 
     fn try_from(l: &EthLog) -> Result<Self, Self::Error> {
         let first_topic = l.topics.first().cloned();
-        let expected_topic = Some(*CANCELLED_USER_OP_TOPIC);
+        let expected_topic = Some(todo!());
```

Comment:
I definitely intend to, although I left it at that temporarily, because it was going to domino into another large commit. I might fit it in before merging if I manage!